server/pom.xml

```diff
@@ -63,7 +63,7 @@
     </dependency>
     <dependency>
      <groupId>org.eclipse.hono</groupId>
-     <artifactId>hono-config</artifactId>
+     <artifactId>hono-demo-certs</artifactId>
```

Comment:
Not sure if we need this anymore. In the past we included the hono-config.jar in order to also have a copy of permissions.json and application.yml etc on the classpath when running tests. We may need to create a permissions.json file in server/src/test/resources for that purpose now ... can you check?